### ISSUE TITLE
App crash on createResizedImage 

### ISSUE THREAD

**Issue by AlaaElden98** [bamlab/react-native-image-resizer#341]
I'm using version `3.0.3` :  `"@bam.tech/react-native-image-resizer": "^3.0.3"`
After capture an image from camera, I'm passing the required data to createResizedImage, the app is crashing on Android -didn't test on ios- with the following error : 
<img src= "https://user-images.githubusercontent.com/34111697/196434849-7bd45435-e6db-4af8-9f66-5690cb28f2bc.png" width= "300" />

Code : 
```
import ImageResizer from '@bam.tech/react-native-image-resizer';

 const result = await ImageResizer.createResizedImage(
        image.uri,
        image.width / 3.8,
        image.height / 3.8,
        'JPEG',
        100,
      );
      if (result) {
            // do stuff
       }
```
all passed params are valid.
Any idea what causes this ?

**Comment by icycommond** [comment]
same for me.

**Comment by ayushmng** [comment]
Same here I'm getting 'createResizedImageWithExceptions' in this issue, has anyone resolved the issue?

**Comment by taboulot** [comment]
@AlaaElden98 

Fix published in 3.0.4.
